ports/wolf3d/Wolfenstein 3D.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+GAMEDIR="/$directory/ports/wolf3d"
+HEIGHT=$DISPLAY_HEIGHT
+WIDTH=$DISPLAY_WIDTH
+
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Create config dir
+rm -rf "$XDG_DATA_HOME/lzwolf"
+rm -rf "$XDG_DATA_HOME/ecwolf"
+ln -s "$GAMEDIR/cfg" "$XDG_DATA_HOME/lzwolf"
+ln -s "$GAMEDIR/cfg" "$XDG_DATA_HOME/ecwolf"
+
+# Permissions
+$ESUDO chmod 666 /dev/tty0
+$ESUDO chmod 666 /dev/tty1
+$ESUDO chmod 666 /dev/uinput
+$ESUDO chmod 777 -R $GAMEDIR/*
+
+# Exports
+export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+export SDL_GAMECONTROLLERCONFIG_FILE="$sdl_controllerconfig"
```

Comment:
What is happening here?
We only export export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"

The SDL_GAMECONTROLLERCONFIG_FILE is set in control.txt to SDL_GAMECONTROLLERCONFIG

Reply:
FILE is used in trimui isn't it? I saw chatter about it being distinct from the former.

Reply:
trimui also has control.txt with the SDL_GAMECONTROLLERCONFIG_FILE pointed to SDL_GAMECONTROLLERCONFIG

Reply:
So what you're saying is it's redundant but harmless.